single-function/lru-cache/lru-cache.test.js

```diff
@@ -0,0 +1,42 @@
+const LRUCache = require('./submission');
```

Comment:
maybe let's call this reference-solution or something that makes it clearer it's the solution?

Reply:
I will do this in another branch for all of them at once.